optuna_dashboard/app.py

```diff
@@ -204,11 +204,19 @@ def delete_study(study_id: int) -> BottleViewReturn:
     @handle_json_api_exception
     def get_study_detail(study_id: int) -> BottleViewReturn:
         response.content_type = "application/json"
+        try:
+            before = int(request.params["before"])
+            assert before >= 0
```

Comment:
@chenghuzi I'm sorry. We should name this `after` because we want to get trials after the given trial number.

Reply:
Fixed in the latest commit